src/panels/lovelace/mixins/element-click-mixin.js

```diff
@@ -12,8 +13,84 @@ import computeStateName from "../../../common/entity/compute_state_name";
 export default dedupingMixin(
   (superClass) =>
     class extends NavigateMixin(EventsMixin(superClass)) {
-      handleClick(hass, config) {
-        const tapAction = config.tap_action || "more-info";
+      registerMouse(config) {
+        let ripple = null;
+        const rippleWrapper = document.createElement("div");
+        this.shadowRoot.appendChild(rippleWrapper);
+        rippleWrapper.style.position = "absolute";
+        rippleWrapper.style.width = "100px";
+        rippleWrapper.style.height = "100px";
+        rippleWrapper.style.top = "50%";
+        rippleWrapper.style.left = "50%";
+        rippleWrapper.style.transform = "translate(-50%, -50%)";
+        rippleWrapper.style.pointerEvents = "none";
```

Comment:
If anyone knows a more handsom way of setting lots of style properties at once I'm all ears.

Reply:
You could create a class and add/remove the class. Or you could use `Object.assign(rippleWrapper.style, {top: "50%", left: "50%", transform: "translate(-50%, -50%)", pointerEvents: "none"})`

Reply:
I would recommend the latter

Reply:
That's what I was looking for.